modules/reference/pages/home/my-organization-configuration.adoc

```diff
@@ -1,7 +1,26 @@
 [[ref.webui.overview.org.config]]
 = Organization Configuration
 
-On the menu:Main Menu[Home > My Organization > Configuration] page modify your personal information, such as name, password, and title.
-To modify any of this information, make the changes in the appropriate text fields and click the btn:[Update] button at the bottom.
+The menu:Home[My Organization > Configuration] section allows you to configure your current organization.
 
-image::home_my_organization_configuration.png[scaledwidth=80%]
+[[org-config-options]]
+.Organization Configuration Options
+[cols="1,1,1", options="header"]
+|===
+| Option                            | Description   | Default
+| Enable staging contents | For clients in this organization, allow content staging by default. | Unchecked
+| Enable Errata E-mail Notifications | For users in this organization, send email notifications when errata is available. | Checked
```

Comment:
is it actually "errata"?  if yes, it's a bug and devs should change it and use "patch".

Reply:
Yeah, I wondered about that, too. It's definitely what the UI says (I checked), but we could change our description to read "patch" instead. Do you think that might be confusing?

Reply:
In those cases, I open an issue/bug with a change request and for the time being "adjust" our text with a parenthesis--I know you do not like these constructs:

> ... send email notifications when errata (patch) is available.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1164649